src/Grocery/AddList.js

```diff
@@ -32,14 +32,41 @@ export default class AddGroceryList extends Component {
 	}
 
 	static navigationOptions({ navigation, screenProps }) {
-		let name = '';
+		let name = 'New Grocery List';
+		let groceryItems = [];
 		const groceryList = screenProps.groceryLists.find(groceryList => groceryList._id === navigation.state.params.id);
 		if (groceryList) {
 			name = groceryList.name;
+			groceryItems = groceryList.items;
 		}
 		return {
-			headerTitle: 'Grocery List',
+			headerTitle: name,
 			headerBackTitle: "Back",
+			headerRight: (
+				<Button 
+					title="Delete"
+					onPress={() => {
+						Meteor.call('grocerylists.remove', navigation.state.params.id, (err) => {
```

Comment:
Should we actually have an 'archive' function for grocery lists as well?

Reply:
yeah I think we should